CMakeLists.txt

```diff
@@ -511,13 +511,24 @@ endif()
 #   - place the CMakeLists.txt in the first-level subdirectory, e.g.
 #     Externals/zlib/CMakeLists.txt (that is: NOT in some Src/ subdirectory)
 #
+include_directories("${CMAKE_CURRENT_BINARY_DIR}/Externals")
```

Comment:
Please don't. Use `target_include_directories` where necessary on a target.

Reply:
It would be nicer if the mgba target provided include directories directly.